app/scripts/controllers/transactions/index.js

```diff
@@ -1502,6 +1509,12 @@ export default class TransactionController extends EventEmitter {
       gasParams.gas_price = gasPrice;
     }
 
+    if (defaultGasEstimates) {
+      gasParams.defaultGasEstimates = this._getGasValuesInGWEI(
```

Comment:
[REQ] This is insufficient because segment won't flatten this object. We'll need to do gasParams.default_gas_price etc.

Reply:
I just updated the PR with fix.